build_tools/travis/install.sh

```diff
@@ -17,6 +17,17 @@ echo 'List files from cached directories'
 echo 'pip:'
 ls $HOME/.cache/pip
 
+if [[ "$DISTRIB" == "pypy" ]]; then
+    ls ~/virtualenv/
+    pip install numpy Cython Tempita
+    # use scipy 1.1 when it's released
+    pip install -vv git+https://github.com/scipy/scipy.git@b3162e187174fa0b1462f7b243c483285b38f45e
```

Comment:
`pip install scipy==1.1.0rc1` may be easier to remember (and maybe gets rid of the Cython timeout, since it's pre-cythonized)...

Reply:
Thanks, fixed! Sorry I missed the RC announcement...